### ISSUE TITLE
Simplify initial configuration

### ISSUE THREAD

**Issue by codeling** [codeling/bfstop#146]
After installation has completed, as a user I want more guidance on enabling and configuring the system.

This might include (on the installation finished screen):
*  A button to enable bfstop with default settings
* A link to directly go to the bfstop settings

**Comment by Goalfair** [comment]
Yes please! Suggestion: either move settings from system plugin settings to component settings or at least add a link in component settings to the system plugin settings. Btw: The inline help switch is nice, perhaps point to it in plugin description or even enable it by default.

**Comment by codeling** [comment]
BFStop is now auto-enabled on installation (https://github.com/codeling/bfstop/commit/26527dd3342cf4e0d7aacf0470bc0a808b1b2ade), and a prominent link is put to the component page containing the consolidated configuration (https://github.com/codeling/bfstop/issues/88)
